packages/flutter/lib/src/cupertino/tableview.dart

```diff
@@ -0,0 +1,485 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/rendering.dart';
+
+@visibleForTesting
+const double kDefaultTableViewCellHeight = 44.0;
+
+@visibleForTesting
+const double kDefaultCellInset = 16.0;
+
+@visibleForTesting
+const double kDefaultDividerHeight = 1.0;
+
+/// WARNING: CupertinoTableView and its associated classes are EXPERIMENTAL. IF
+/// YOU USE THESE APIS, WE WILL BREAK YOU!
+///
+/// An iOS-styled table view.
+///
+/// Table views are used in iOS to render lists of cells. They are the iOS
+/// equivalent of Android's `ListView` and `RecyclerView`. [CupertinoTableView]
+/// is Flutter's iOS-styled alternative to [ListView].
+///
```

Comment:
Add one sentence before going into details below about what's different / why would one use a cupertino table vs a list view.

Reply:
My thought was that it would be obvious for all Cupertino widgets that the answer to that is "if you want iOS styling, use the Cupertino thing".  Do you think we require some kind of explicit treatment here?

Reply:
I think so. We should strive to make Flutter an easy thing to let people build anything on a spectrum than making it too much of a exclusive binary choice. People who don't do this all the time might not what the iOS thing is. If they're on the fence about whether to use Material or Cupertino or mix in between or create something custom, knowing what they're choosing between lets them make better choices.

Reply:
Ok, more verbiage added.